src/task2.java

```diff
@@ -0,0 +1,98 @@
+public class task2 {
+    public task2() {
+    }
+
+    static void stringToArrayToArrayToString() {
```

Comment:
java method names typically follow camelCase convention
for example, stringToArrayToArrayToString could be renamed to stringToArrayAndBackToString